packages/rum-core/test/createFakeClick.ts

```diff
@@ -12,7 +12,7 @@ export function createFakeClick({
 }: {
   hasError?: boolean
   hasPageActivity?: boolean
-  userActivity?: { selection?: boolean; input?: boolean }
+  userActivity?: { selection?: boolean; input?: boolean; scroll?: boolean }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ 

```suggestion
  userActivity?: Partial<UserActivity>
```

Reply:
Done in https://github.com/DataDog/browser-sdk/pull/2669/commits/e2d1a5d15b7f638efb080b74c00b60449658e32d